kolibri/plugins/user_profile/assets/src/machines/changeFacilityMachine.js

```diff
@@ -74,6 +74,7 @@ const setMerging = assign({
 export const changeFacilityMachine = createMachine({
   id: 'machine',
   initial: 'selectFacility',
+  predictableActionArguments: true,
```

Comment:
Could you please explain what was the reason for this change?

Reply:
https://xstate.js.org/docs/guides/actions.html#actions

Reply:
it was recommended by the xstate library in the browser warnings, you can see it in the screenshot you attached at https://github.com/learningequality/kolibri/pull/9584#pullrequestreview-1093031427